includes_php71/class-wcmp-frontend-settings.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+if ( ! defined('ABSPATH')) exit;  // Exit if accessed directly
+
+if ( ! class_exists('WooCommerce_MyParcelBE_Frontend_Settings')) :
+
+/**
+ * @deprecated
+ * Frontend settings
+ */
+class WooCommerce_MyParcelBE_Frontend_Settings {
+
+    const DAYS_SATURDAY = 6;
+    const CARRIER_CODE = 2;
+    const CARRIER_NAME = "Bpost";
+    const BASE_URL = "https://api.myparcel.nl/";
+
+    /**
+     * @var \WPO\WC\MyParcelBE\Collections\SettingsCollection
+     */
+    private static $settings;
+
+    function __construct() {
+        exit ('dit gaat weg als dat kan');
```

Comment:
Hier staat nog een exit, kijk of dit inderdaad weg kan

Reply:
done